be/src/exec/pipeline/sink/mysql_table_sink_operator.cpp

```diff
@@ -79,13 +79,17 @@ void MysqlTableSinkIOBuffer::close(RuntimeState* state) {
 }
 
 void MysqlTableSinkIOBuffer::_process_chunk(bthread::TaskIterator<ChunkPtr>& iter) {
-    --_num_pending_chunks;
+    DeferOp op([&]() {
+        --_num_pending_chunks;
```

Comment:
what if stop here and continue later, is it possible that sink_io_buffer destroyed in the gap?